src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -32,7 +32,7 @@ const DiscoveryTab = ({onChangeScreen, selectedScreen = 0, tabs}) => {
     <View style={styles.tabContainer}>
       <ScrollView
         showsVerticalScrollIndicator={false}
-        howsHorizontalScrollIndicator={false}
+        showsHorizontalScrollIndicator={false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `howsHorizontalScrollIndicator={false}` to `showsHorizontalScrollIndicator={false}` is a good fix as the previous code was incorrect and would have caused an error. The ScrollView component does not recognize the prop `howsHorizontalScrollIndicator`. The correct prop is `showsHorizontalScrollIndicator`, which controls whether to show the horizontal scroll indicator.
```diff
-        howsHorizontalScrollIndicator={false}
+        showsHorizontalScrollIndicator={false}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->